packages/transactions/src/signatures.ts

```diff
@@ -77,10 +77,16 @@ export function assertTransactionIsFullySigned<TTransaction extends CompilableTr
         .map(a => a.address);
     const requiredSigners = new Set([transaction.feePayer, ...signerAddressesFromInstructions]);
 
+    const missingSigs: Address[] = [];
     requiredSigners.forEach(address => {
         if (!transaction.signatures[address]) {
-            // TODO coded error
-            throw new Error(`Transaction is missing signature for address \`${address}\``);
+            missingSigs.push(address);
         }
     });
+
+    if (missingSigs.length > 0) {
+        const startErr = missingSigs.length === 1 ? 'signature for address' : 'signatures for addresses';
+        // TODO coded error
+        throw new Error('Transaction is missing ' + startErr + ': ' + `${missingSigs.map(a => `'${a}'`).join(', ')}`);
+    }
```

Comment:
Nice! I'm wondering what you think about just simplifying this and always throwing:

```
Transaction is missing signatures for addresses:
```

In the case that there's only one, it's just a list of length one, no biggie.

Also, just a minor nit: can we get rid of the single quotes or ticks altogether? Just printing the address with no quotes/markers I think is OK.